lintcheck/src/main.rs

```diff
@@ -555,9 +581,15 @@ fn main() {
     build_clippy();
     println!("Done compiling");
 
+    let cargo_clippy_path = fs::canonicalize(format!("target/debug/cargo-clippy{EXE_SUFFIX}")).unwrap();
+    let clippy_driver_path = fs::canonicalize(format!("target/debug/clippy-driver{EXE_SUFFIX}")).unwrap();
+
```

Comment:
this makes me wonder if we should actually offer a --release mode somehow :sweat_smile:

Reply:
That could be cool, I wonder how much you need to lint before it makes sense to apply 😄